### DISCUSSION TITLE
How to find out the theme token of one specific part of a component?

### DISCUSSION THREAD

**Discussion by see311** [ant-design/ant-design#40773] [Q&A]
How to find out the theme token of one specific part of a component?
For example, which token of background color is used when mouse hover over a row of table?

**Comment by see311** [comment]
I've found the color token of 'tableRowHoverBg', called 'colorFillAlterSolid'. How to modify this token in a formal way?

**Comment by gariggio** [comment]
I love antd, but the new theming system used since version 5.x is very bad.
Wanting to change a particular stylistic aspect of a specific theme-level component is a nightmare.
Most of the tokens suggested by IDE don't work when used at theme/component level and it's impossible to figure out which tokens are available and which aren't.
Would it be possible at least to have a list of working tokens that can be changed component by component?
I will be grateful to anyone who can change my mind...

**Comment by abenhamdine** [comment]
You'll probably have to wait for https://github.com/ant-design/ant-design/issues/41884 to be completed

**Comment by gariggio** [comment]
I just noticed that at the bottom of each antd component's official documentation is the "Design Tokens" section.
I haven't done any checking yet but if the tokens used by component are well documented and if they work, then **antd theming** is finally usable... :-)
